app/code/Magento/Directory/Setup/UpgradeData.php

```diff
@@ -149,6 +152,49 @@ private function getDataForAustralia()
         ];
     }
 
+    /**
+     * Mexican states data.
+     *
+     * @return array
+     */
+    private function getDataForMexico()
+    {
+        return [
```

Comment:
I comared the region list with the wiki list and looks like "Mexico City" (CMX) region is missing https://en.wikipedia.org/wiki/Template:Mexico_State-Abbreviation_Codes However, I'm not sure if it should be included